src/software/ai/hl/stp/tactic/chip/chip_tactic.h

```diff
@@ -69,3 +69,5 @@ class ChipTactic : public Tactic
     // Tactic parameters
     ChipFSM::ControlParams control_params;
 };
+
```

Comment:
nit: plz add a comment here explaining what this does

Reply:
Fixed